### ISSUE TITLE
Some kubernetes requirements missing from v0.25

### ISSUE THREAD

**Issue by afbjorklund** [boot2podman/boot2podman#20]
Apparently I had forgotten to include "pinns" and "ipset" requirements

So that means that crio and k3s won't start, but is fixed on master now

**Comment by afbjorklund** [comment]
Also need to install "conntrack" bin

`The binary conntrack is not installed, this can cause failures in network connection cleanup.`

And to upgrade the coredns version.

```
docker.io/coredns/coredns                  1.6.3     c4d3d16fe508   4 months ago    44.4 MB
docker.io/coredns/coredns                  1.3.0     2ee68ed074c6   13 months ago   40.6 MB
```

**Comment by afbjorklund** [comment]
Fixed in `v0.26`
